web/pdf_find_controller.js

```diff
@@ -377,3 +407,60 @@ var PDFFindController = (function PDFFindControllerClosure() {
   return PDFFindController;
 })();
 
+var PDFURLFinder = (function PDFFindBarClosure() {
+    function PDFURLFinder(page) {
+        try {
+            var paramname = 'search';
+
+            //search parameter contains search terms- 
+            //multiple space (%20) separated words or phrases
+            //search term syntax: a b c single words,
+            //a=b c=d e=f=g phrases
+
+            //check local window for search terms
+            //make sure to check for pound symbol
+            var loc = document.location.href.replace('#', '&');
+            loc = loc.replace(/\+/ig, '%20');
+            var params = PDFView.parseQueryString(loc);
+
+            //also check parent window for any search terms
+            if (params[paramname] === undefined) {
+                loc = parent.document.location.href.replace('#', '&');
+                loc = loc.replace(/\+/ig, '%20');
+                params = PDFView.parseQueryString(loc);
+            }
+
+            if (params[paramname] !== undefined) {
+                var parambase = params[paramname].replace(/\"/ig, '');
+                PDFJS.multiple = parambase.split(/\s+/);
```

Comment:
Maybe consider a better name than `multiple`.... multiple what? Maybe a better one would be `multipleSearchParam`?